Libplanet.Store.Remote/Extensions/KeyValueStoreExtensions.cs

```diff
@@ -0,0 +1,24 @@
+using Libplanet.Store.Trie;
+
+namespace Libplanet.Store.Remote.Extensions
+{
+    public static class KeyValueStoreExtensions
+    {
+        public static bool TryGet(this IKeyValueStore store, KeyBytes key, out byte[] value)
+        {
+            bool result;
+            try
+            {
+                value = store.Get(key);
+                result = true;
+            }
+            catch
+            {
+                value = Array.Empty<byte>();
+                result = false;
+            }
+
+            return result;
+        }
+    }
+}
```

Comment:
This single use extension seems unnecessary. I think it should be part of `RemoteKeyValueService.Get()`.

Reply:
I don't wanna popping up data logic to the controller. I want to leave `RemoteKeyValueService.Get()` just a controller.

Reply:
The way I see it, throwing an `Exception` is baked in to the model layer, as it is well documented in `IKeyValueStore` interface. If we look at it that way, as `Exception` could be considered as a valid "response" from the data model layer, I would say it is the controller's job to `catch` and convert that `Exception` to a serviceable response. It definitely isn't part of the current data model. 

One way to have so called "popping data logic" abstracted away would be to create a new `interface` or `class` that wraps the base `IKeyValueStore`, and pretend `RemoteKeyValueService` is to serve that abstracted model, but that would be an overkill and confusing. 🙄

I understand wanting to have the controller devoid of logic as much as possible, but I don't think having the transformation of the model behavior into a response just "floating around" as an extension method is necessarily a good idea.

Reply:
Check this method used in `RemoteKeyValueService`, not `RemoteKeyValueStore`. When throwing `Exception` on `RemoteKeyValueService.Get()` is a problem of the server response, not `IKeyValueStore`.

But yes, I understand what you said "floating around" means. I'll move this method into `RemoteKeyValueService`.

Reply:
https://github.com/planetarium/libplanet/pull/3688/commits/9a4d4e08ee39a9a55f18346b34b1db5fa9ae3bd8 amended.